### PR TITLE
JAV-105 create microservice from definition

### PR DESCRIPTION
**Pull request by wujimin**

### CODE REVIEW

File: service-registry/src/main/java/io/servicecomb/serviceregistry/RegistryUtils.java [hunk 1]
```diff
@@ -61,6 +60,9 @@
 public final class RegistryUtils {
     private static final Logger LOGGER = LoggerFactory.getLogger(RegistryUtils.class);
 
+    // temp field
+    private static MicroserviceManager microserviceManager;
```
Comment (WillemJiang) [inline service-registry/src/main/java/io/servicecomb/serviceregistry/RegistryUtils.java]: It's not thread safe if there are multiple thread init the microserviceManager.

**Comment by wujimin** [comment]
create microservice from microservice definition, and save them in MicroserviceManager

File: service-registry/src/test/java/io/servicecomb/serviceregistry/config/TestPropertiesLoader.java [hunk 2]
```diff
@@ -16,31 +16,31 @@
...
+import io.servicecomb.config.archaius.sources.ConfigModel;
+import io.servicecomb.config.archaius.sources.MicroserviceConfigLoader;
+import io.servicecomb.serviceregistry.api.registry.Microservice;
+import io.servicecomb.serviceregistry.api.registry.MicroserviceInstance;
+import io.servicecomb.serviceregistry.api.registry.MicroserviceManager;
+import io.servicecomb.serviceregistry.definition.DefinitionConst;
+import io.servicecomb.serviceregistry.definition.MicroserviceDefinition;
 
 public class TestPropertiesLoader {
+    private static MicroserviceManager microserviceManager = new MicroserviceManager();;
```
Comment (WillemJiang) [inline service-registry/src/test/java/io/servicecomb/serviceregistry/config/TestPropertiesLoader.java]: ;;

File: service-registry/src/main/java/io/servicecomb/serviceregistry/RegistryUtils.java (hunk 1 above)
Comment (wujimin) [inline service-registry/src/main/java/io/servicecomb/serviceregistry/RegistryUtils.java]: only be inited when cse start, and only init once.

Comment (wujimin) [inline service-registry/src/main/java/io/servicecomb/serviceregistry/RegistryUtils.java]: and all about this field code is temp
all will be changed after the follow PR.

**Pull request merged**